<launch>
  <!-- We use sim time so that time starts from 0 and not from the walltime. This means playabck should start immediately  -->
  <param name="/use_sim_time" value="false"/>
  <param name="out_file" value="$(arg out_file)"/>
  <param name="video_stream_namespace" type="string" value="/camera/color/" />
  <param name="fx" value="1000.0" />
  <param name="baseline" value="0.09" />
  <param name="depth_units" value="0.001" />
  <node pkg="vdo_bag_generation" type="usyd_dataset_pc_collect_node" name="usyd_dataset_pc_collect" output="screen" />
</launch>